bats/tests/helpers/vm.bash

```diff
@@ -9,19 +9,22 @@ assert_rd_is_stopped() {
     # run ps -ef
     # local $rancherdesktop="/Applications/Rancher Desktop.app/"
     # ! [[ $output =~ $rancherdesktop ]]
-    osascript -e 'if application "Rancher Desktop" is running then error 1' 2>/dev/null
+    is_macos && osascript -e 'if application "Rancher Desktop" is running then error 1' 2>/dev/null
+    is_linux && ! pgrep rancher-desktop &>/dev/null
 }
 
 wait_for_shutdown() {
     try --max 18 --delay 5 assert_rd_is_stopped
 }
 
 factory_reset() {
-    run $RDCTL shutdown
+    $RDCTL shutdown
```

Comment:
Why did you remove the call to `run`. I put it there so the function doesn't fail if RD already isn't running anymore.

Reply:
You are right, reverting back.